camviewer_ui_impl.py

```diff
@@ -9,63 +9,57 @@
 #
 from camviewer_ui import Ui_MainWindow
 from psp.Pv import Pv
-#from xtcrdr import xtcrdr
 from dialogs import advdialog
 from dialogs import markerdialog
 from dialogs import specificdialog
-from dialogs import dropletdialog
-from dialogs import xtcrdrdialog
 from dialogs import timeoutdialog
 from dialogs import forcedialog
 
+import sys, os
+sys.path.append(os.getcwd() + "/pycaqtimage")
```

Comment:
This seems unstable

Reply:
Yeah.  Added __init__.py to pycaqtimage and then from pycaqtimage import pycaqtimage, and life is good.

Or better, anyway.